samples/subsys/shell/shell_module/prj.conf

```diff
@@ -9,7 +9,8 @@ CONFIG_THREAD_MONITOR=y
 CONFIG_BOOT_BANNER=n
 CONFIG_THREAD_NAME=y
 CONFIG_DEVICE_SHELL=y
-CONFIG_POSIX_CLOCK=y
+CONFIG_POSIX_API=y
```

Comment:
With these changes, these tests fail to build for native_posix[//64], so they will fail in main's CI.
Please either ensure they keep working, or filter these platforms out.

Reply:
Certainly!

I'll wait to push changes until later, just to ensure that other change requests have been captured as well.

Reply:
Fixed!